cypress/integration/11:workflow/transitions/1:create.feature

```diff
@@ -10,3 +10,18 @@ Feature: CREATE: Transition on workflow
     When I can see "Statuses & transitions" text on "title-bar-header" element
     * On "workflow-designer" I can see column with "New" value
     * On "workflow-designer" I can see column with "Draft" value
+    * On "workflow-designer" I can see column with "status_EN" value
+
+  @other
+  Scenario: Add workflow transition - give up
+    When On "workflow-designer" in row 2 I create transition from "New" to "status_EN"
+    * On "workflow-designer" I click on the "remove" button for the transition from "New" to "status_EN"
+    Then I confirm modal
+
+  @success
+  Scenario: Add workflow transition - success
```

Comment:
Creating multiple transitions? From New => status_EN and from Draft => New ? Isn't one transition enough?

Reply:
yes, one enough